modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/FormURL.es.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import FormURL from '../../../src/main/resources/META-INF/resources/admin/js/util/FormURL.es';
+
+const formInstanceId = 1;
+
+describe('FormURL', () => {
+	it('gets a blank url', () => {
+		const published = false;
+		const requireAuthentication = false;
```

Comment:
@marcosapmf, does it need to be `const` or we can use `let`, for example?

Reply:
@natocesarrego, it does not need to `const` but `formatSource` updates that